cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,83 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if billingAccount == "" && !interactive.Enabled() {
+		billingAccount = aws.GetJumpAccount("local")
+		r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+		r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+	}
+
+	if interactive.Enabled() {
+		cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+
+		if len(cloudAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  cloudAccounts,
+			})
+
+			billingAccount = ocm.RegexpReturnOnlyInteger(billingAccount)
+
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid billing account: '%s'", err)
+				os.Exit(1)
+			}
+		}
+
+		if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
+			r.Reporter.Errorf("Expected a valid billing account")
+			os.Exit(1)
+		} else {
+			format := "Jan 02, 2006"
+			// Get contract info
+			if billingAccount != awsCreator.AccountID {
+				r.Reporter.Infof("The selected AWS billing account is a different account than your AWS infrastructure account." +
+					"The AWS billing account will be charged for subscription usage. " +
+					"The AWS infrastructure account will be used for managing the cluster. \n" +
+					"   Contract enabled for this billing account. \n")
+			} else {
+				r.Reporter.Infof("Using '%s' as billing account. \n"+
+					"   Contract enabled for this billing account. \n",
+					billingAccount)
+			}
+			contracts, err := r.OCMClient.GetBillingAccountContracts(billingAccount)
+
+			if err != nil {
+				r.Reporter.Errorf("%s", err)
+				os.Exit(1)
+			}
+			for _, contract := range contracts {
+				dimensions := contract.Dimensions()
+				r.Reporter.Infof(
+					"+---------------------+----------------+ \n"+
+						"   | Start Date          |'%s'  | \n"+
+						"   | End Date            |'%s'  | \n"+
+						"   +---------------------+----------------+ \n",
+					contract.StartDate().Format(format), contract.EndDate().Format(format))
```

Comment:
we could probably move this to its own function `printContracts`

Reply:
The output will also look a little different from the mockup - we probably want to show start date, end date, number of cpus and cluster in the same "block"